### PR TITLE
I395 dates replaced by ActiveTriples::Relations

### PR DESCRIPTION
**Pull request by ShanaLMoore**
# Story

Refs #395  

# Expected Behavior Before Changes
When a user edited a Journal Article and saved it, some fields would change to display ActiveTriple::Relations strings.

# Expected Behavior After Changes
On Edit, the data persists as expected. Date fields should render as dates, for example. 

# Screenshots / Video

<details> 

<summary>

**BEFORE**

</summary>

image

</details>

<details> 

<summary>

**AFTER**

</summary>

image

</details>

# Notes

If this does not resolve production/staging data, we may need to reimport those works. 

**TODO**: Client to verify and make a list of ones she notices aren't corrected. 

## ADDITIONAL CONTEXT
ref: https://github.com/scientist-softserv/adventist-dl/pull/398#issuecomment-1542281641

### CODE REVIEW

**Review by jeremyf — APPROVED** [review]

File: app/views/journal_articles/edit_fields/_default.html.temp.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<% # the hyrax version of this file messes singular field display %>
```
Comment (alishaevn) [inline app/views/journal_articles/edit_fields/_default.html.temp.erb:1]: can you state that this is an override file, and list which version of hyrax you're overriding?

File: app/views/records/edit_fields/_default.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<% # the hyrax version of this file messes singular field display %>
```
Comment (alishaevn) [inline app/views/records/edit_fields/_default.html.erb:1]: can you state that this is an override file, and list which version of hyrax you're overriding?

File: app/views/hyrax/journal_articles/edit_fields/_date_created.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<% # use date picker %>
```
Comment (alishaevn) [inline app/views/hyrax/journal_articles/edit_fields/_date_created.html.erb:1]: can you state that this is an override file, and list which version of hyrax you're overriding?

File: app/views/journal_articles/edit_fields/_date_created.html.erb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<% # use date picker %>
```
Comment (alishaevn) [inline app/views/journal_articles/edit_fields/_date_created.html.erb:1]: can you state that this is an override file, and list which version of hyrax you're overriding?

**Review by alishaevn — CHANGES_REQUESTED** [review]

**Comment by ShanaLMoore** [comment]
@alishaevn Thanks for your review but these aren't overrides. These partials come from [dog biscuit](https://github.com/samvera-labs/dog_biscuits/blob/main/lib/generators/dog_biscuits/templates/app/views/records/edit_fields/_default.html.erb). It should have been installed in this application when the dog biscuit generator was run. We discovered that most of them were but some were removed along the way, when the aark_id work was implemented, so we are putting them back. 

I'm aware that they may have been removed for a reason, but so far I've tested other work types and I haven't noticed any changes regarding the aark_id.

**Review by alishaevn — APPROVED** [review]

File: app/views/records/edit_fields/_default.html.erb (hunk 2 above)
Comment (ShanaLMoore) [reply app/views/records/edit_fields/_default.html.erb:1]: ref: https://github.com/scientist-softserv/adventist-dl/pull/398#issuecomment-1542281641

File: app/views/journal_articles/edit_fields/_default.html.temp.erb (hunk 1 above)
Comment (ShanaLMoore) [reply app/views/journal_articles/edit_fields/_default.html.temp.erb:1]: ref: https://github.com/scientist-softserv/adventist-dl/pull/398#issuecomment-1542281641

**Pull request merged**